package.json

```diff
@@ -1,17 +1,19 @@
 {
   "name": "tailwindcss-intellisense",
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
   "devDependencies": {
     "@npmcli/package-json": "^5.0.0",
+    "@types/culori": "^2.1.0",
+    "culori": "^4.0.1",
+    "esbuild": "^0.20.2",
+    "minimist": "^1.2.8",
```

Comment:
I don't really like that these deps have to be at the root package level but it's happening because the shared esbuild.mjs file is at the root level rather than having an exact copy inside each package.